### ISSUE TITLE
Videos at 60 fps, problem with subtitles

### ISSUE THREAD

**Issue by Astalavista1655** [elishacloud/Silent-Hill-2-Enhancements#886]
Hello everyone, since the game now supports 60 FPS without bugs, I interpolated the videos at 60 FPS and converted them to .bik, the game accepts the files, there is no audio out of sync, but there is subtitle out of sync. I offer the project ready-made videos, but I don’t know what to do with subtitles, can the developers adapt them to the videos? The weight of all videos is 8.51 GB, if necessary I will re-encode for a lower bitrate (weight is simply not important to me, the main thing is quality)
https://www.youtube.com/watch?v=NmC8izu0gKQ

**Comment by Polymega** [comment]
Hi there,

We made a teaser announcement for this [over on our Twitter page](https://twitter.com/WhereAllBegins/status/1693294746901315903), but the next project update will include 60 FPS FMVs along with properly synced subtitles. Thank you though for reaching out about this!

**Comment by Astalavista1655** [comment]
> Hi there,
> 
> We made a teaser announcement for this [over on our Twitter page](https://twitter.com/WhereAllBegins/status/1693294746901315903), but the next project update will include 60 FPS FMVs along with properly synced subtitles. Thank you though for reaching out about this!
Did you do the interpolation in After Effects? Hehe, I'll try again) Try the RIFE 2.3 neural network (Flowframes program, where scene changes are automatically corrected), it is much better than AE. Will I be able to replace it with my own files while maintaining the correct synchronization of subtitles?

**Comment by Astalavista1655** [comment]
> Hi there,
> 
> We made a teaser announcement for this [over on our Twitter page](https://twitter.com/WhereAllBegins/status/1693294746901315903), but the next project update will include 60 FPS FMVs along with properly synced subtitles. Thank you though for reaching out about this!

Did you do the interpolation in After Effects? Hehe, I'll try again) Try the RIFE 2.3 neural network (Flowframes program, where scene changes are automatically corrected), it is much better than AE. Will I be able to replace it with my own files while maintaining the correct synchronization of subtitles?

**Comment by Polymega** [comment]
Yep, I used Flowframes to make the FMVs 60 FPS. I keep my entire workflow lossless up until the time to render in Bink. So to review any work done (such as using Flowframes), I import it into After Effects for preview due to using a lossless PNG image sequence for my working files.

When we release our next project update, you should be able to use your own files without problem.

Thanks!

**Comment by Astalavista1655** [comment]
> Yep, I used Flowframes to make the FMVs 60 FPS. I keep my entire workflow lossless up until the time to render in Bink. So to review any work done (such as using Flowframes), I import it into After Effects for preview due to using a lossless PNG image sequence for my working files.
> 
> When we release our next project update, you should be able to use your own files without problem.
> 
> Thanks!

Ha, not bad, in this case there is no need to use my own files
